public/main.js

```diff
@@ -0,0 +1,32 @@
+const clickEvent = function (event) {
+    const id = event.target.getAttribute('data-question-id');
+    const helpful = JSON.parse(event.target.getAttribute('data-helpful'));
```

Comment:
Do you need the `JSON.parse()` here?

Reply:
yes, I need JSON.parse() to change the string to boolean

Reply:
:+1: